lib/services/summary_service.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Aprb.Service.SummaryService do
+  alias Aprb.{Repo, Topic, Summary}
+
+  def update_summary(topic, event) do
+    current_date = Calendar.Date.today! "America/New_York"
+    cond do
+      Enum.member?(~w(subscriptions users inquiries purchases), topic) ->
+        handle_summary(topic, event["verb"], current_date)
+      topic == "bidding" ->
```

Comment:
Ugh yeah I can imagine special cases like this littering all the services attempting to consume these events.

Since it's still early in our usage, maybe we can rationalize the 2 forms into a at least _some_ common structure? If there are properties in earlier events' schemas that don't make sense as we expand into bidding, they can maybe be dropped from the shared schema and included in the additional data. There must be _some_ common data.